crates/core/src/test_utils/constants.rs

```diff
@@ -9,4 +9,6 @@ pub const EVM_CONTRACTS: &[&str] = &["ERC20", "Counter", "PlainOpcodes"];
 lazy_static! {
     pub static ref EOA_WALLET: LocalWallet = EOA_PRIVATE_KEY.parse().unwrap();
     pub static ref DEPLOY_FEE: FieldElement = FieldElement::from(100000u32);
```

Comment:
Can we remove this since you added the same constant in `crates/core/src/client/constants.rs`

Reply:
done